### DISCUSSION TITLE
I would like to update recipes to gulp v4 and add them to the docs

### DISCUSSION THREAD

**Discussion by silvenon** [gulpjs/gulp#2495] [Ideas] [answered]
I think that adding them to the docs is crucial for discoverability because I keep forgetting that these recipes exist. Then we would be able to tag them and add a search bar. I think it's an excellent way for people to learn gulp.

Would adding it to the docs be welcome? As "Recipes" in the navigation, after "Plugins"?

**Comment by phated** [comment, marked as answer]
@silvenon We actually have an issue about rewriting all the recipes for v4! https://github.com/gulpjs/gulp/issues/2164

Basically, when we wrote the v4 docs, I actually went through the recipes to update the syntax and noticed that the quality of patterns being recommended was very low, so I believe we need to throw them all out and start from scratch.

What do you think? Would you like to take that on?

**Reply by silvenon** [reply]
Yes! 💪
